examples/loco_starter/src/models/_entities/files.rs

```diff
@@ -0,0 +1,31 @@
+//! `SeaORM` Entity. Generated by sea-orm-codegen 0.12.4
```

Comment:
We should use the 1.0 codegen

Reply:
I removed these lines https://github.com/SeaQL/sea-orm/pull/2230/commits/fea1725b8bee19acd97920895652c384e3d24acf